src/api/v1/invites/handlers/create-invite-code.ts

```
@@ -0,0 +1,106 @@
+import { InviteCodeStatus } from "@prisma/client";
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)